cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -146,7 +146,7 @@ func (k *Kuberhealthy) Start(ctx context.Context) {
 	go k.stateReflector.Start()
 
 	// if influxdb is enabled, configure it
-	if enableInflux {
+	if cfg.EnableInflux == false {
```

Comment:
This would turn on influx if EnableInflux is false no?

Reply:
That is correct...I have pushed an update that checks for true....I am not 100% positive why I did that at the time